src/main/flight/interpolated_setpoint.c

```diff
@@ -58,9 +77,36 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float rxInterval = currentRxRefreshRate * 1e-6f;
         const float rxRate = 1.0f / rxInterval;
 
-        const float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
-        const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
+        float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
+        float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]);
+        const float holdSteps = 2.0f;
+
+        // Glitch reduction code for identical packets
+        if ((setpointSpeed == 0) && (fabsf(rawSetpoint) < 0.98f * ffMaxRate[axis])) {
```

Comment:
Unneeded braces.

Reply:
thanks

Reply:
Ah OK it was the inner braces.  Removed.